vignettes/ggPMX-guide.Rmd

```diff
@@ -467,6 +532,20 @@ ctr <- pmx_mlx(
 
 ```
 
+### Models fitted with NONMEM (versions 7.2 and later) 
+
+It is required to provide simulation tables to generate VPCs. Furthermore, it is highly recommended that simulation tables have a "sim"-suffix and are kept with the same naming convetion as for the prediction tables (e.g sdtab001sim, catab001sim)). In this case they're automatically recognized by the runnumber (runno) or 
+by the model file if specified there. For post-hoc simulation it is possible to include an additional simfile:
+
+```{r, eval = FALSE}
+ctr <- pmx_nm(
+  directory = model_dir,
+  file      = "modelfile.ctl" #or .lst
+  simfile   = "simulation_modelfile.ctl" #or .lst
+)
+```
+
+
```

Comment:
Probably need a nlmixr section here;  It should be easy enough to add.

Reply:
Yes! It was on the todo list, but forgot to add it, now its in there!